src/Templates/src/templates/maui-mobile/Pages/Controls/AddButton.xaml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Button xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
```

Comment:
I'm wondering if creating a `Style` instead of a custom control for this would be better